### PR TITLE
Refactor analysis for per turn eval

### PR DESCRIPTION
**Pull request by mus1cholic**
**Patch description**
Refactors code for processing left_pane_text so code injections can be made on other crowdsourcing tasks

**Testing steps**
See CircleCI tests

### CODE REVIEW

File: parlai/crowdsourcing/tasks/acute_eval/webapp/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5938 @@
+{
```
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/acute_eval/webapp/package-lock.json]: This is a generated file - we don't want to include it in the PR

File: parlai/crowdsourcing/tasks/model_chat/conf/example.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ mephisto:
     include_persona: false
     conversation_start_mode: 'hi'
     annotation_question: Does this comment from your partner have any of the following attributes? (Check all that apply)
-    conversations_needed_string: "blender_90M:10"
+    conversations_needed_string: "blender_3B:2"
```
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/conf/example.yaml]: let's keep the example the same - we want the casual user of model chat to not have to load the large Blender model

File: parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py [hunk 3]
```diff
@@ -253,6 +251,11 @@ def __init__(
             }
         )
 
+    def process_left_pane_text(self, args: "DictConfig") -> str:
```
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py:254]: hmm, honestly, I think that these 3 lines of code will probably be the same for all subclasses of model_chat, meaning that the lines will need to get repeated in all subclasses. I think it'll lead to less duplicated code if, instead, there's a method called something like `format_left_pane_text()`, which you just define as `pass` for this superclass

File: parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml [hunk 4]
```diff
@@ -1,2 +1,6 @@
-blender_90M: >
-    --model-file zoo:blender/blender_90M/model
+#blender_90M: >
```
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml]: Good to not add in these commented-out lines =P

**Review by EricMichaelSmith — COMMENTED** [review]
Hi Orion! This is a great idea - but I think there's a slightly different setup that will lead to even less code-reuse :)

File: parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml [hunk 5]
```diff
@@ -1,2 +1,3 @@
 blender_90M: >
     --model-file zoo:blender/blender_90M/model
+
```
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml:3]: Nit: extra space

**Review by stephenroller — COMMENTED** [review]
I concur with Eric's comments

**Review by stephenroller — APPROVED** [review]

File: parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml (hunk 5 above)
Comment (mus1cholic) [reply parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml]: Resolved

File: parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py (hunk 3 above)
Comment (mus1cholic) [reply parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py]: Resolved

**Review by EricMichaelSmith — APPROVED** [review]
Looks good, thanks! But make sure to wait until all CI checks pass before merging in :)

**Comment by mus1cholic** [comment]
CI tests passed, merging in now

**Pull request merged**